packages/react-components/review/beta/react-components.api.md

```diff
@@ -66,6 +67,55 @@ export interface BaseCustomStyles {
     root?: IStyle;
 }
 
+// @beta
+export const BrowserPermissionDenied: (props: BrowserPermissionDeniedProps) => JSX.Element;
+
+// @beta
+export const BrowserPermissionDeniedIOS: (props: BrowserPermissionDeniedIOSProps) => JSX.Element;
+
+// @beta
+export interface BrowserPermissionDeniedIOSProps extends BrowserPermissionDeniedProps {
+    imageSource?: string;
```

Comment:
Would there be value in adding a comment here saying where this image would go/ an example of them using it?

Reply:
Good point, I'll add a short comment about it.